CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 ### Fixes
 
+- [Pull request #1745: Update vendor polyfills to match upstream](https://github.com/alphagov/govuk-frontend/pull/1745).
```

Comment:
Do we think this is useful for users to have this in the release notes, as it's mostly an internal change?

Reply:
I err on the side of strict most of the time, we'd probably get away with not having an entry though. Up to you.

Reply:
I have no particular preference, was just unsure about it myself. Maybe @m-green can look at this when we come to cut the next release.